docs/guide/opentelemetry-elastic.asciidoc

```diff
@@ -92,63 +96,159 @@ You can also build the collector-contrib repository by cloning it and running:
 make otelcontribcol
 ----
 
-[float]
-[[open-telemetry-elastic-configure]]
-==== Configure the collector
+[[open-telemetry-elastic-traces-metrics]]
+==== Collect traces and metrics
 
-Create a `yaml` configuration file.
+To export traces and metrics to the OpenTelemetry Collector, ensure that you have instrumented your services and applications
+with the OpenTelemetry API and/or SDK. If you haven't, see the Elastic APM <<install-and-run,install and run guide>> to get started
+with Elastic APM Agents instead.
```

Comment:
I like the examples below, but I think we should be even more clear here that the main purpose of this guide is OTel integration with Elastic, not OTel agent setup. What if we added a NOTE to the top of this section, and a link to OTel api/sdk setup at the bottom?

---

NOTE: This guide assumes your services and applications have already been instrumented with the OpenTelemetry API and/or SDK. If you're new to APM, we recommend <<install-and-run,getting started with Elastic APM agents>> instead.

To export traces and metrics to the OpenTelemetry Collector, ensure that you have instrumented your services and applications with the OpenTelemetry API and/or SDK.

Here is an example of how to set up the OpenTelemetry Java agent.

// example

Here is an example of how to capture business metrics from an application.

// example

For more information on setting up OTel, see <link>.

---

Reply:
Great suggestion. Thanks!